app/_kong_plugins/statsd/examples/log-on-status-range.yaml

```
@@ -0,0 +1,26 @@
+description: 'Log requests in a status range '
+
+title: 'Enable Metrics'
+
+weight: 900
+
+requirements:
+ - "[Service](/gateway/entities/service)"
```

```suggestion
 - "A [Gateway Service](/gateway/entities/service) configured"
```
I wasn't sure why this was a requirement even though you can configure it globally.